.travis.yml

```diff
@@ -15,16 +16,19 @@ install:
   - export GIT_FULL_HASH=`git rev-parse HEAD`
   - conda config --set always_yes true
   - conda update conda
-  - conda create -n testenv pip nose python=$TRAVIS_PYTHON_VERSION numpy scipy=0.15.1 scikit-image six coverage
-  - conda install -n testenv -c scikit-xray xraylib lmfit=0.8.3 netcdf4
+  - conda config --add channels scikit-xray
```

Comment:
I preferred the clarify of two lines, telling me which packages come from us and which come from Continuum.